core/src/main/scala/org/genivi/sota/core/WebService.scala

```diff
@@ -106,6 +110,36 @@ class VehiclesResource(db: Database, rviClient: RviClient)
         rviClient.sendMessage(s"genivi.org/vin/${vin}/sota/getpackages", io.circe.Json.Empty, ttl())
         // TODO: Confirm getpackages in progress to vehicle?
         complete(NoContent)
+      } ~
+      pathPrefix("updates") {
```

Comment:
What do you mean? Move the new routes to another class? Because all this is already under `api/v1/vehicles`, so what should be moved to the new class?